assets/src/js/listing_pages.js

```diff
@@ -1,4 +1,5 @@
 export const setupListingPages = () => {
+  // Setup behaviour for list/grid toggle.
```

Comment:
Thinking loud: should we wait for the DOM to be loaded before executing the code within the `setupListingPages` function?

Reply:
We can, I tried adding it actually and it didn't seem to change anything, I know we do it for some of these functions and for others we don't (by "these functions" I mean the ones similar to this one, that we add via `app.js`)